packages/core/faststore.config.default.js

```diff
@@ -99,4 +99,16 @@ module.exports = {
     enableCypressExtension: false,
     noRobots: false,
   },
+  rewrites: async function rewrites() {
+    return {
+      beforeFiles: [
+        {
+          source: '/test/my-office',
+          destination: `/office`,
```

Comment:
```suggestion
          destination: '/office',
```

Reply:
This is just a temporary code for testing purposes that will be removed before the merge.